docs/_includes/install/universal.sh

```diff
@@ -49,18 +49,28 @@ then
     exit 1
 fi
 
+clickhouse_download_filename_prefix="clickhouse"
+clickhouse="$clickhouse_download_filename_prefix"
+
+i=0
+while [ -f "$clickhouse" ]
+do
+    clickhouse="${clickhouse_download_filename_prefix}.${i}"
+    i=$(($i+1))
```

Comment:
Is it bash-only or compatible with Mac OS and FreeBSD shells?

PS. Nevertheless, ok to merge, because the typical scenario will work.

Reply:
all expressions (except `i=$(($i+1))` - which is quite trivial I think) are used in this script, so, I guess, it's compatible the same as it was...